{"name": "Негативный тест авторизации пользователя, неверные логин и пароль.", "status": "passed", "description": "Отправляем запрос авторизации с данными, которых нет в системе.    Ожидаемый статус код 401.    Ожидаемый ответ запроса содержит \"success\" is False.", "start": 1739192792840, "stop": 1739192792945, "uuid": "56265dd1-5675-4dab-9d10-4f37688dedd7", "historyId": "fcc5e002bc5c4fb2da15a9d69fee8cec", "testCaseId": "fcc5e002bc5c4fb2da15a9d69fee8cec", "fullName": "tests.test_auth_user.TestAuth#test_auth_invalid_data_user_code_401", "labels": [{"name": "suite", "value": "Логин пользователя."}, {"name": "parentSuite", "value": "tests"}, {"name": "subSuite", "value": "TestAuth"}, {"name": "host", "value": "MacBook-Pro-Katrin.local"}, {"name": "thread", "value": "33116-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_auth_user"}]}